### PR TITLE
before after 

### PR DESCRIPTION
**Pull request by erol-g**
before after image comparison slider is added.

### CODE REVIEW

File: app/_components/BeforeAfter.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import Image from "next/image";
+import React, { useState } from "react";
+
+const BeforeAfter = () => {
+	const [sliderPosition, setSliderPosition] = useState(50);
+	const [isDragging, setIsDragging] = useState(false);
+
```
Comment (muhsinaltintop) [inline app/_components/BeforeAfter.jsx:9]: unnecessary enter

File: app/_components/BeforeAfter.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		const percent = Math.max(0, Math.min((x / rect.width) * 100, 100));
+		setSliderPosition(percent);
+	};
+	const handleMouseDown = () => {
+		setIsDragging(true);
+	};
+	const handleMouseUp = () => {
+		setIsDragging(false);
+	};
+
```
Comment (muhsinaltintop) [inline app/_components/BeforeAfter.jsx:23]: unnecessary enter

File: next.config.mjs [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
-    domains: ["pics.freeicons.io"], // Add your allowed image domains here
-  },
+	images: {
+		remotePatterns: [
+			{
+				protocol: "https",
+				hostname: "pics.freeicons.io",
+				port: "",
+			},
+
```
Comment (muhsinaltintop) [inline next.config.mjs:10]: unnecessary enter

File: next.config.mjs [hunk 4]
```diff
@@ -1,8 +1,26 @@
...
+			{
+				protocol: "https",
+				hostname: "**.hearstapps.com",
+				port: "",
+			},
+
```
Comment (muhsinaltintop) [inline next.config.mjs:16]: unnecessary enter

**Review by muhsinaltintop — COMMENTED** [review]
There are unnecessary enters otherwise looks good to me.

**Review by elifalinli — APPROVED** [review]
lgtm after other suggested small fixes

**Pull request merged**